<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="columns">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="train">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>530</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Train</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Root Mean Sq. Error:</string>
    </property>
   </widget>
   <widget class="QLabel" name="target">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Dropout or Graduate&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mean Square Error:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conf_mat">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>480</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Confusion Matrix</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="test_data">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>410</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Target : </string>
    </property>
   </widget>
   <widget class="QLabel" name="rmse">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mean Absolute Error:</string>
    </property>
   </widget>
   <widget class="QLabel" name="mae">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="test_size_btn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Test Data Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="mse">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="accuracy">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Accuracy Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="test_size">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>490</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Test Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="train_size">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Train Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Classification Report:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="report">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>451</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="predict">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>530</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Predict</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Light</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(11, 34, 57);</string>
    </property>
    <property name="text">
     <string>Logistic Regression</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
